docker/package/model.py

```diff
@@ -323,10 +323,12 @@ def gen_spec_file(self, build_deps, run_deps, out):
 make {binary_name}
 mkdir -p %{{buildroot}}/%{{_bindir}}
 install -m 0755 {binary_name} %{{buildroot}}/%{{_bindir}}
+ln -s %{{_bindir}}/{binary_name} %{{buildroot}}/%{{_bindir}}/{self.name}
```

Comment:
These work alright, but symlinks for `tezos-node-<protocol>` don't, as we discussed probably because they are not listed under `%files`.

For the time being let's add the `--force` to their `ln` invocation in the node post-installation

Reply:
the problem is not inside that line

Reply:
I guess it's a good idea to do `-sf` here as well, since this symlinking can be done for some subsequent version as well, and thus `ln -s` during the installation may fail since `/usr/bin/{self.name}` already exists

Reply:
force pushed since commit needed reword in presence of this suggestion